docs/resources/hook.md

```diff
@@ -33,6 +33,11 @@ EOF
   dependencies = {
     auth0 = "2.30.0"
   }
+
+  secrets = {
+    secretKey1 = secretValue1
+    secretKey2 = secretValue2
+  }
```

Comment:
Looks like this is defined already in lines [29-31](https://github.com/alexkappa/terraform-provider-auth0/pull/391/files#diff-ddfac45a2e6738941b3aec8a6064be2f0e577048a4a259920f8106feefb4639cR29-R31). Is that not sufficient?

Reply:
Ah, yes, apologies, didn't have latest checked out. Will close.